web_list_html_widget/__openerp__.py

```diff
@@ -6,14 +6,13 @@
     "version": "8.0.1.0.0",
     "depends": ["web"],
     'author': "initOS GmbH & Co. KG, Sunflower IT, "
-              "Odoo Community Association (OCA),",
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/web"
```

Comment:
@hbrunn Now fixed, plus flake8 errors.

Reply:
thanks!

Reply:
hmmm, check the diff on github, it seems not to be actually fixed